code/modules/mob/living/living.dm

```diff
@@ -291,17 +291,36 @@
 	if(ismob(AM))
 		var/mob/M = AM
 
+
 		log_combat(src, M, "grabbed", addition="passive grab")
 		if(!supress_message && !(iscarbon(AM) && HAS_TRAIT(src, TRAIT_STRONG_GRABBER))) //Everything in this if statement handles chat messages for grabbing
 			var/mob/living/L = M
-			if (L.getorgan(/obj/item/organ/tail) && zone_selected == BODY_ZONE_PRECISE_GROIN) //Does the target have a tail?
-				M.visible_message("<span class ='warning'>[src] grabs [L] by [L.p_their()] tail!</span>",\
-								"<span class='warning'> [src] grabs you by the tail!</span>", null, null, src) //Message sent to area, Message sent to grabbee
-				to_chat(src, "<span class='notice'>You grab [L] by [L.p_their()] tail!</span>")  //Message sent to grabber
-			else
-				M.visible_message("<span class='warning'>[src] grabs [M] [(zone_selected == BODY_ZONE_L_ARM || zone_selected == BODY_ZONE_R_ARM)? "by their hands":"passively"]!</span>", \
-								"<span class='warning'>[src] grabs you [(zone_selected == BODY_ZONE_L_ARM || zone_selected == BODY_ZONE_R_ARM)? "by your hands":"passively"]!</span>", null, null, src) //Message sent to area, Message sent to grabbee
-				to_chat(src, "<span class='notice'>You grab [M] [(zone_selected == BODY_ZONE_L_ARM|| zone_selected == BODY_ZONE_R_ARM)? "by their hands":"passively"]!</span>") //Message sent to grabber
+			var/grab_sanity_check = FALSE
+
+			switch(zone_selected)
+				if(BODY_ZONE_HEAD)
+					if(ishuman(L))//Special handling required for checking clothing
+						var/mob/living/carbon/human/H = L
+						if (istype(H.wear_neck, /obj/item/clothing/neck/petcollar))
+							grab_output(src, L, "collar", GRAB_KILL)
+							//L.apply_damage(5, OXY)
+							setGrabState(GRAB_KILL)
+							log_combat(src, M, "grabbed", addition="kill grab")
+							grab_sanity_check = TRUE
+					if(!grab_sanity_check)
+						grab_output(src, L, "head", GRAB_PASSIVE)
+				if(BODY_ZONE_L_ARM)
+					grab_output(src, L, "arm", GRAB_PASSIVE)
+				if(BODY_ZONE_R_ARM)
+					grab_output(src, L, "arm", GRAB_PASSIVE)
+				if(BODY_ZONE_PRECISE_GROIN)
+					if(L.getorgan(/obj/item/organ/tail))
+						grab_output(src, L, "tail", GRAB_PASSIVE)
+					else
+						grab_output(src, L)
+				else
+					grab_output(src, L, "passively", GRAB_PASSIVE)
```

Comment:
*Person grabs other by their passively*
*Person grabs you by the passively!*

Reply:
Identified this bug during Golden 3 playtesting. Already fixed in an upcoming commit.